nx.json

```diff
@@ -29,6 +29,12 @@
         "target": "build",
         "projects": "dependencies"
       }
+    ],
+    "docker": [
+      {
+        "target": "build",
+        "projects": "self"
+      }
```

Comment:
This is optional, without this you need to execute the nx build target before the docker target because you are copying the compiled assets in the dockerfile: `COPY ./dist/apps/hackathon-api .`